Include/genobject.h

```diff
@@ -45,6 +45,12 @@ PyAPI_FUNC(PyObject *) _PyGen_Send(PyGenObject *, PyObject *);
 PyObject *_PyGen_yf(PyGenObject *);
 PyAPI_FUNC(void) _PyGen_Finalize(PyObject *self);
 
+#define PYGEN_RETURN 0
+#define PYGEN_ERROR -1
+#define PYGEN_NEXT 1
+
+PyAPI_FUNC(int) PyGen_Send(PyGenObject *, PyObject *, PyObject **);
```

Comment:
I'd like you to add some docs here (as a comment) and to the actual C API docs. You'll also need to update `refcounts.dat`